aptos-move/framework/aptos-framework/sources/jwks.move

```diff
@@ -201,6 +201,28 @@ module aptos_framework::jwks {
         assert!(num_bytes < MAX_FEDERATED_JWKS_SIZE_BYTES, error::invalid_argument(EFEDERATED_JWKS_TOO_LARGE));
     }
 
+    // Todo: description
+    public fun update_federated_jwk_set(jwk_owner: &signer, iss: vector<u8>, kid_vec: vector<String>, alg_vec: vector<String>, e_vec: vector<String>, n_vec: vector<String>) acquires FederatedJWKs {
```

Comment:
Is this necessary? Why are you adding it?

Reply:
Discussed offline.  Can be done via script but given this is a common usecase and scripts require move compilation it is good to have it in the framework